internal/engine/pod.go

```diff
@@ -40,6 +41,11 @@ func handlePodChangeAction(ctx context.Context, state *store.EngineState, action
 
 	oldRestartTotal := podInfo.AllContainerRestarts()
 	podInfo.Containers = podContainers(ctx, pod)
+	if isNew {
```

Comment:
this block read pretty confusingly to me: "if isNew" followed by "we want to do stuff for existing pods" while "new" and "existing" sound pretty antonym-y

maybe something like "// we want to discount any restarts that occurred before we first saw the pod"?

Reply:
i tried to expand the comment a bit